Makefile

```diff
@@ -1,21 +1,18 @@
 LICENSE := ASL2-Short
 VERSION ?= local
 
-check-fmt: goimports go-licenser
-	@go-licenser -d -license ${LICENSE}
-	@goimports -l -e -local github.com/elastic . | read && echo "Code differs from gofmt's style. Run 'gofmt -w .'" 1>&2 && exit 1 || true
+GOIMPORTS := go run golang.org/x/tools/cmd/goimports@latest
+GOLICENSER := go run github.com/elastic/go-licenser@latest
```

Comment:
Suggest that if `@latest` are used, the makefile emits the version of the tools during the build.
```
curl -s https://proxy.golang.org/golang.org/x/tools/@latest | jq '.Version'
curl -s https://proxy.golang.org/github.com/elastic/go-licenser/@latest | jq '.Version'
```

Reply:
I think what you are saying is that we should not use `@latest`? Based on recent pains, I would agree. Moving toward a more hermetic build would be better despite the increased effort to periodically update versions.

Reply:
I would prefer not to have dynamic versions, yes.

Reply:
I opened https://github.com/elastic/stream/pull/80 to automate this.